doc/api_assets/style.css

```
@@ -718,10 +799,24 @@ kbd {
   }
 }
 
+.header {
+  position: relative;
+  padding-top: 1rem;
+  background-color: var(--color-fill-app);
+}
+
+@media only screen and (min-height: 1001px) {
+  .header {
+    z-index: 1;
+    top: -1px;
+    position: sticky;
+  }
+}
```

Suggestion: maybe we should have sticky "by default" since that will also be turned on with the JS version
```suggestion
.header {
  position: sticky;
  top: -1px;
  z-index: 1;
  padding-top: 1rem;
  background-color: var(--color-fill-app);
}

@media not screen, (max-height: 1000px) {
  .header {
    position: relative;
    top: 0;
  }
}
```